src/vs/workbench/common/editor/editorStacksModel.ts

```diff
@@ -944,6 +944,16 @@ export class EditorStacksModel implements IEditorStacksModel {
 		return this._groups.indexOf(group);
 	}
 
+	public findGroup(editorInput: EditorInput, activeOnly: Boolean): EditorGroup {
+		const found = this.groups.filter(group =>
+			group.getEditors().some(editor => {
+				const isActive = group.isActive(editor);
+				return editor.matches(editorInput)
```

Comment:
You should first check for the `isActive` condition and then do the `editor.matches` call to avoid it unless needed.